cloud_experts_tutorials/cloud-experts-gitops-with-argocd.adoc

```diff
@@ -0,0 +1,181 @@
+// aliases: ['/experts/demos/gitops/']
+// tags: ["GitOps", "ROSA", "ARO"]
+// authors:
+//   - Steve Mirman
+//   - Connor Wooley
+// ---
+
+The purpose of this document is to help you get  OpenShift GitOps running in your cluster, including deploying a sample application and demonstrating how ArgoCD ensures environment consistency.
+
+[NOTE]
+====
+If you prefer visual guide, you can watch a quick start video on link:https://www.youtube.com/watch?v=Gi18iemF1yI[YouTube].
+====
+
+== Prerequisites
+
+* You have a Managed OpenShift Cluster with cluster-admin rights.
+* You have the following tools installed:
+    ** link:https://git-scm.com/download/[git]
+    ** link:https://docs.openshift.com/container-platform/4.13/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli_cli-developer-commands[oc]
+    ** link:https://argoproj.github.io/argo-cd/cli_installation/[argocd]
+    ** link:https://kubectl.docs.kubernetes.io/installation/kustomize/[kam]
+
+== Environment Set Up
+
+1. Install the OpenShift GitOps Operator from the **Operator Hub**:
++
+image::gitops_operator.png[screenshot of GitOps install]
+
+2. Clone the `gitops-demo` GitHub repository to your local machine by running the following command:
++
+[source,terminal]
+----
+$ git clone https://github.com/rh-mobb/gitops-demo gitops
+----
+
+3. Export your local path to the GitHub files:
++
+[source,terminal]
+----
+$ export GITOPS_HOME="$(pwd)/gitops"
+$ cd $GITOPS_HOME
+----
+
+4. Log in to OpenShift by copying the login command from the OpenShift console:
++
+image::oc_login.png[screenshot of login]
+
+5. Enter the command in your terminal to authenticate with the OpenShift CLI (oc).
++
+.Sample output
++
+[source,terminal]
+----
+Logged into "https://<YOUR-INSTANCE>.openshiftapps.com:6443" as "<YOUR-ID>" using the token provided.
+----
+
+== Deploy the ArgoCD Project
+
+1. Create a new OpenShift project called *gitops* by running the following command:
++
+[source,terminal]
+----
+$ oc new-project gitops
+----
+
+2. Add **cluster-admin** rights to the `openshift-gitops-argocd-application-controller` service account in the **openshift-gitops** namespace by using the following command:
++
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user cluster-admin -z openshift-gitops-argocd-application-controller -n openshift-gitops
+----
+
+3. Retrieve the ArgoCD URL to log into your ArgoCD:
++
+[source,terminal]
+----
+$ argoURL=$(oc get route openshift-gitops-server -n openshift-gitops -o jsonpath='{.spec.host}{"\n"}')
+$ echo $argoURL
+----
+
+4. Retrieve ArgoCD Password:
++
+[source,terminal]
+----
+$ argoPass=$(oc get secret/openshift-gitops-cluster -n openshift-gitops -o jsonpath='{.data.admin\.password}' | base64 -d)
+$ echo $argoPass
+----
+
+5. In a web browser, navigate to the ArgoCD console using the `$argoURL` value:
++
+image::argo1.png[screenshot of argocd1]
+
+6. Log in with the user name **admin** and the password returned as `$argoPass`:
```

Comment:
```suggestion
. Log in with the username *admin* and the password returned as `$argoPass`:
```
I'm not sure why the username is bold and the password is in code block here. I would think it would be one or the other?